neo/test/iotest/test_cedio.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from neo.io import CedIO
+from neo.test.iotest.common_io_test import BaseTestIO
+
+
+class TestCedIO(BaseTestIO, unittest.TestCase, ):
+    ioclass = CedIO
+    entities_to_test = [
+        'spike2/m365_1sec.smrx'
```

Comment:
Is there also an `smr` file available for running the tests on?

Reply:
yep I can add it.

Reply:
Done